dev/bots/flutter_build_test.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+readonly SCRIPTS_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+readonly ROOT_DIR="$SCRIPTS_DIR/.."
+
+function is_expected_failure() {
+  # A test target was specified with the 'build' command.
+  grep --quiet "is not configured for Running" "$1"
+}
+
+# Navigate to codelab and attempt to build 104-complete
+echo "git clone https://github.com/material-components/material-components-flutter-codelabs.git"
```

Comment:
A lot of duplicated strings for commands. Any way they can be local variables instead to prevent them from ever getting out of sync?